tracer/propagation.go

```diff
@@ -0,0 +1,232 @@
+package tracer
+
+import (
+	"encoding/binary"
+	"io"
+	"strconv"
+	"strings"
+
+	"github.com/gogo/protobuf/proto"
+	opentracing "github.com/opentracing/opentracing-go"
+	"github.com/wavefronthq/wavefront-opentracing-sdk-go/wire"
+)
+
+const (
+	prefixTracerState = "ot-tracer-"
```

Comment:
Let's be consistent with our other SDKs: https://github.com/wavefrontHQ/wavefront-opentracing-sdk-java/blob/c90317d3450c672e48a6bbb165e67b36adf01422/src/main/java/com/wavefront/opentracing/propagation/TextMapPropagator.java#L21

This is to ensure that all our SDKs inject/extract using the same keys so everything works cross-language across process boundaries.

Reply:
right... I need to try that.